testing/fixture_test.h

```diff
@@ -8,6 +8,7 @@
 #include <memory>
 
 #include "flutter/common/settings.h"
+#include "flutter/fml/mapping.h"
```

Comment:
Nit: Unsure why this include was necessary. ELFAOTSymbols is not in fml/mapping.